auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.java

```diff
@@ -335,25 +337,38 @@ public AuthenticationRequest loginWithOAuthAccessToken(@NonNull String token, @N
      * }
      * </pre>
      *
-     * @param phoneNumber      where the user received the verification code
-     * @param verificationCode sent by Auth0 via SMS
-     * @param connection       to end the passwordless authentication on
+     * @param phoneNumber       where the user received the verification code
+     * @param verificationCode  sent by Auth0 via SMS
+     * @param realmOrConnection to end the passwordless authentication on
      * @return a request to configure and start that will yield {@link Credentials}
      */
     @SuppressWarnings("WeakerAccess")
-    public AuthenticationRequest loginWithPhoneNumber(@NonNull String phoneNumber, @NonNull String verificationCode, @NonNull String connection) {
-        Map<String, Object> parameters = ParameterBuilder.newAuthenticationBuilder()
-                .set(USERNAME_KEY, phoneNumber)
-                .set(PASSWORD_KEY, verificationCode)
-                .setGrantType(GRANT_TYPE_PASSWORD)
+    public AuthenticationRequest loginWithPhoneNumber(@NonNull String phoneNumber, @NonNull String verificationCode, @NonNull String realmOrConnection) {
```

Comment:
So to clarify, this doesn't change the public signature? The parameter name is just for the variable used inside the function.

Reply:
Correct. The public signature remains the same.